client/tests/acceptance/user-can-login-test.js

```diff
@@ -37,4 +40,24 @@ module('Acceptance | user can login', function(hooks) {
     assert.equal(currentURL(), '/logout');
     assert.equal(currentSession().isAuthenticated, false);
   });
+
+  test('User sees error message if no CRM contact is found for their email', async function (assert) {
+    const accessToken = 'a-valid-jwt';
+    this.server.create('user');
+    this.server.get('/login', () => new Response(401, { some: 'header' }, {
+      errors: [{
+        response: {
+          code: 'NO_CONTACT_FOUND',
+          message: 'This message is nice but does not affect frontend logic',
```

Comment:
This message _is_ nice, if useless 🤣

Reply:
Loooove that y'all caught this. Makes me happy.